### ISSUE TITLE
getOrder with RDT

### ISSUE THREAD

**Issue by mbolder** [fond-of-vertigo/amazon-sp-api#48]
Hello, since we moved to v.0.7.1 we also wanted to updated the order handling. 
There we ran into the following problem. 
After creating a RDT and filter with:

`
rr := &tokens.CreateRestrictedDataTokenRequest{
        RestrictedResources: []tokens.RestrictedResource{
	        {
		        Method:       http.MethodGet,
		        Path:         "/orders/v0/orders",
		        DataElements: []string{"buyerInfo", "shippingAddress"},
	        },
        },
        }
tokenResp, err := client.TokenAPI.CreateRestrictedDataTokenRequest(rr)
log.Infof("Fetched RDT=%s", tokenResp.ResponseBody.RestrictedDataToken)
rdt := *tokenResp.ResponseBody.RestrictedDataToken
no := time.Now()
fro := no.Add(time.Hour * 24 * -10)
from := apis.JsonTimeISO8601{Time: fro}
filter := &orders.GetOrdersFilter{CreateAfter: from, MarketplaceIDs: []constants.MarketplaceID{constants.Germany}}
response, err := client.OrdersAPI.GetOrders(filter, &rdt)
`
Our response is 400, the RDT response is correct but the filter seems to be wrong. 
Any ideas what I am doing wrong here?

**Comment by Coffeeri** [comment]
Hey there,
I have reproduced your error.
This is unfortunately once again an amazon bug.
https://github.com/amzn/selling-partner-api-models/issues/2687

The error I produced:
```
{
  "errors": [
    {
      "code": "InvalidInput",
      "message": "timestamp must follow ISO8601",
      "details": ""
    }
  ]
}
```
However our time format is compliant with the one in their documentation.
https://developer-docs.amazon.com/sp-api/docs/iso-8601

The first link to the bug mentions a fix, by using the UTC timezone. This worked for me as well. I add this shortly as a quick fix.
